src/page/forside/gjennomforing-liste/gjennomforing-panel/GjennomforingPanel.tsx

```diff
@@ -31,6 +36,18 @@ export const GjennomforingPanel = ({ gjennomforing }: GjennomforingPanelProps):
 							{gjennomforing.opprettetAar}/{gjennomforing.lopenr}
 						</BodyShort>
 					</div>
+					<div className={styles.info}>
+						{
+							avsluttet &&
+							<div className={styles.tags}>
```

Comment:
Kan ikke se at stylingen for tags er tilgjengelig

Reply:
Den ligger her..? https://github.com/navikt/amt-tiltaksansvarlig-flate/blob/avsluttetstatus/src/page/forside/gjennomforing-liste/gjennomforing-panel/GjennomforingPanel.module.scss#L18

Reply:
Kommenterte visst på feil fil jeg, det var den andre som mangla styling

Reply:
oj, det så jeg ikke, good catch! 🏆

Reply:
Da jeg flyttet dette ut i en egen komponent forsvant hele problemstilingen 🙃